package.json

```diff
@@ -28,6 +28,7 @@
   },
   "scripts": {
     "lint": "npm run lint-registry && npm run lint-cli",
+    "lint-check": "prettier --check '**/*.js'",
```

Comment:
Should this get chained into the `lint` target?

Reply:
Hmm, that seems like it could be helpful. If I did, I think I would embed it into `lint-cli` and `lint-registry`, but I saw this as project-wide linting specifically for the CI.

I'm not really sure what's ergonomically preferable. Maybe keeping `lint-check` in the root for project-wide linting in the CI, and then tacking `prettier --check` onto both `lint-cli` and `lint-registry`?

Reply:
Not sure, I submitted #134 and figured that if this added the general `lint` it would cover both cases :smile:
I'm not sure if the preference in the project is for subpackage targets or not